### ISSUE TITLE
[CHORE] - Run Prettier on the repo

### ISSUE THREAD

**Issue by kcoderhtml** [thepurplebubble/site#58]
### What work needs to be performed?

run `bunx prettier --write .`

**Comment by ofluffydev** [comment]
I added prettier to the server-node repo, does it need to be added to the site or just ran?

**Comment by kcoderhtml** [comment]
Just ran; adding it as a GitHub workflow for prs would be awesome though

**Comment by ofluffydev** [comment]
Yeah I talked to jasper about the workflow addition, maybe change this todo to be that instead?
